Dockerfile

```diff
@@ -3,5 +3,6 @@ FROM registry.opensource.zalan.do/stups/openjdk:latest
 EXPOSE 8086
 
 COPY target/zmon-data-service-1.0-SNAPSHOT.jar /zmon-data-service.jar
+COPY target/scm-source.json /
```

Comment:
``scm-source.json`` should not be included in opensource image.

Reply:
See above comment about how the Dockerfile is no longer compatible with other build systems.  It is no harm to include it in a CDP image as CDP will ignore it.  It is not possible to deploy a non-CDP-built image without this.

Reply:
scm-source is Z specific and should not be in opensource image - regardless of the build system.

Reply:
Anything supporting CDP is Z specific as well.  And you'll find scm-sources all over our opensource images as CDP is relatively new and not so widely used yet for all scenarios.  When CDP gets further along (supports more build workflows) we can align more on what in-principle is correct.

Reply:
having scm-source in built image is not a problem. Having the requirement to run scm-source before building is the problem and Z specific.